### DISCUSSION TITLE
Timer for timebox is required on all steps. Hide input informations for collect

### DISCUSSION THREAD

**Discussion by wesleyzilva** [microsoft/vsts-extension-retrospectives#207] [Ideas]
![image](https://user-images.githubusercontent.com/55301153/120489044-ae7a7c80-c38d-11eb-9441-1dfb01b93247.png)

> TIMER FOR TIMEBOX IS REQUIRED ON ALL STEPS

> HIDE INFORMATIONS WHEN IS COLLECTING OR VOTING

**Comment by nigeledw** [comment]
A countdown timer is required which the facilitator can set. It should be a general timer available in the right top corner that everyone can see.
It should not be specific to any process in the Retro as facilitators may run retros in different ways.

So, for example the facilitator can let everyone know we have a 5-minute timed exercise to just identify what went well first. Then we have discussion, grouping and voting. Then we go back and repeat the process starting a timed exercise again on what didn't go well.
